### ISSUE TITLE
Is database-sampler used during pretraining?

### ISSUE THREAD

**Issue by FrontierBreaker** [chaytonmin/Voxel-MAE#25]

**Comment by FrontierBreaker** [comment]
I previously think the proposed method is an unsupervised one. However, if gt-sampler is used during pretraining, the proposed method is actually not. Could you tell me whether I get the point? Thanks!

**Comment by FrontierBreaker** [comment]
@chaytonmin

**Comment by chaytonmin** [comment]
> @chaytonmin

This is just the data process of OpenPCDet.

**Comment by FrontierBreaker** [comment]
So, you mean database-sampler is used for Voxel-MAE pretraining?

Actually, when I run the pretraining script, I find that DB-sampler is used.

**Comment by chaytonmin** [comment]
> So, you mean database-sampler is used for Voxel-MAE pretraining?
> 
> Actually, when I run the pretraining script, I find that DB-sampler is used.

Our occupancy labels are generated by the input voxels, which is not related to the gt boxes. No matter what data sample strategy is used.

**Comment by FrontierBreaker** [comment]
Got that, thanks!!

**Comment by FrontierBreaker** [comment]
To the best of knowledge, I think the database sampler introduced in voxelmae somehow indicates that the training data is more than pure lidar&image pairs. Instead, It is the combination of  lidar&image  and pre-generated object database.

**Comment by chaytonmin** [comment]
> To the best of knowledge, I think the database sampler introduced in voxelmae somehow indicates that the training data is more than pure lidar&image pairs. Instead, It is the combination of lidar&image and pre-generated object database.

Yes, it will include the pre-generated object database. However, it does not include the box information in the occupancy labels.

**Comment by FrontierBreaker** [comment]
Great thanks to your patient feedbacks! : )
